### PR TITLE
Introduce code coverage reporting

### PR DESCRIPTION
**Pull request by willkill07**

### CODE REVIEW

**Review by DavitPetro — DISMISSED** [review]
Looks good.

**Comment by willkill07** [comment]
I really don't think this should be merged. It needs to exclude the `code/unit_tests` directory. Plus, it's somehow not including any other python file located out of `code/models/`

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[run]
+omit =
+    tests/*
```
Comment (16aburnett) [inline .coveragerc:3]: Should this be `unit-tests/*` instead of `tests/*` since our tests are in the unit-tests/ directory?

Comment (jdchica2020) [reply .coveragerc:3]: I will fix that. Thank you!

**Comment by 16aburnett** [comment]
I worked with Joan and added the updated coveragerc file. It now omits the unit-test directory but despite including other files, the report still only shows for files in code/models/ 

Is it possible that the files don't show because they aren't hit at all but the unit-tests?

**Review by jdchica2020 — COMMENTED** [review]
I have the same issue. I can only show code/models. I did the report too and it's the same.Thank you.
Screen Shot 2020-11-13 at 2 10 55 AM

**Comment by willkill07** [comment]
After some investigation, it looks like source can be used instead of include:

https://coverage.readthedocs.io/en/coverage-5.1/source.html#source

@16aburnett @jdchica2020

**Comment by 16aburnett** [comment]
I changed the include to source and now it tracks the other files. I also had to remove the asterisks because they kept giving warnings and prevented a report from being made.

**Review by willkill07 — COMMENTED** [review]
@DavitPetro if you re-review this should be good to go

**Review by 16aburnett — APPROVED** [review]

**Pull request merged**